manifests/12_prometheus_rule.yaml

```
@@ -8,11 +8,59 @@ metadata:
+          expr: sum(app_registry_request_total{code=~"4..",opsrc="community-operators"})
+        - record: app_registry:community_operators:5xx_response
+          expr: sum(app_registry_request_total{code=~"5..",opsrc="community-operators"})
+        - alert: CommunityOperatorConnectionErrors
+          annotations:
+            message: Unable to connect to the default OperatorSource community-operators for {{ $value }}% of requests.
+          expr: increase(app_registry_request_total{code!~"2..",opsrc="community-operators"}[10m]) / increase(app_registry_request_total{opsrc="community-operators"}[10m]) > 0.5
+          for: 10m
+          labels:
+            severity: warning
+    - name: marketplace.certified_operators.rules
+      rules:
+        - record: app_registry:certified_operators:1xx_response
+          expr: sum(app_registry_request_total{code=~"1..",opsrc="certified-operators"})
+        - record: app_registry:certified_operators:2xx_response
+          expr: sum(app_registry_request_total{code=~"2..",opsrc="certified-operators"})
+        - record: app_registry:certified_operators:3xx_response
+          expr: sum(app_registry_request_total{code=~"3..",opsrc="certified-operators"})
+        - record: app_registry:certified_operators:4xx_response
+          expr: sum(app_registry_request_total{code=~"4..",opsrc="certified-operators"})
+        - record: app_registry:certified_operators:5xx_response
+          expr: sum(app_registry_request_total{code=~"5..",opsrc="certified-operators"})
+        - alert: CertifiedOperatorConnectionErrors
+          annotations:
+            message: Unable to connect to the default OperatorSource certified-operators for {{ $value }}% of requests.
+          expr: increase(app_registry_request_total{code!~"2..",opsrc="certified-operators"}[10m]) / increase(app_registry_request_total{opsrc="certified-operators"}[10m]) O > 0.5
+          for: 10m
+          labels:
+            severity: warning
+    - name: marketplace.redhat_operators.rules
+      rules:
+        - record: app_registry:redhat_operators:1xx_response
+          expr: sum(app_registry_request_total{code=~"1..",opsrc="redhat-operators"})
+        - record: app_registry:redhat_operators:2xx_response
+          expr: sum(app_registry_request_total{code=~"2..",opsrc="redhat-operators"})
+        - record: app_registry:redhat_operators:3xx_response
+          expr: sum(app_registry_request_total{code=~"3..",opsrc="redhat-operators"})
+        - record: app_registry:redhat_operators:4xx_response
+          expr: sum(app_registry_request_total{code=~"4..",opsrc="redhat-operators"})
+        - record: app_registry:redhat_operators:5xx_response
+          expr: sum(app_registry_request_total{code=~"5..",opsrc="redhat-operators"})
+        - alert: RedhatOperatorConnectionErrors
+          annotations:
+            message: Unable to connect to the default OperatorSource redhat-operators for {{ $value }}% of requests.
```

I feel like registry is missing here?
```suggestion
            message: Unable to connect to the default OperatorSource redhat-operators registry for {{ $value }}% of requests.
```

---

Same for all 3 msgs.